domain/operation/state/types.go

```diff
@@ -0,0 +1,100 @@
+// Copyright 2025 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package state
+
+import (
+	"database/sql"
+	"time"
+)
+
```

Comment:
todo: remove

- operationRow
- operationTaskRow
- operationUnitTaskRow
- operationMachineTaskRow
- operationTaskStatusRow

the structures appear to be unused.